Providers/Pocket/PocketFeed.swift

```diff
@@ -6,7 +6,24 @@ import Foundation
 import Shared
 import Storage
 
-class Pocket: FeatureFlaggable, URLCaching {
+protocol PocketStoriesProviding {
+    typealias StoryResult = Swift.Result<[PocketFeedStory], Error>
```

Comment:
What `Swift.` doing here? I've never seen that before.

Reply:
So using `Swift.` is telling the compiler you want to use the class from the standard library. For example if you would have your own implementation of `Result` type named exactly the same, the compiler would use that implementation and not the Foundation one. This was the problem with the migration and could be easily fixed by referring to it with that prefix. Now, I assume in the past Firefox had its own implementation and it was discarded so it's probably not needed anymore. I carried it from `typealias ContileResult = Swift.Result<[Contile], Error>
`